src/mlpack/methods/ann/layer/concat.hpp

```diff
@@ -194,6 +207,15 @@ class Concat
   void serialize(Archive& /* ar */, const unsigned int /* version */);
 
  private:
+  //! Parameter which indicates the input size of modules.
+  std::vector<int> inputSize;
```

Comment:
I'd recommend using an `Armadillo` vector here.

Reply:
Sure, I'll add it.